src/index.js

```diff
@@ -1,12 +1,66 @@
-if (process.env.NODE_ENV === 'development') {
-  const { worker } = require('./mocks/browser');
-  worker.start();
-}
+/* ß */
+
+//DOM selectors
+const form = document.getElementById('search-form');
+const input = document.getElementById('search-input');
+const blankDetails = document.getElementById('empty-pokemon-details');
+const filledDetails = document.getElementById('filled-pokemon-details');
 
 function getPokemon(name) {
   axios
     .get('https://pokeapi.co/api/v2/pokemon/' + name)
-    .then((res) => console.log(res.data))
+    .then((res) => {
+      //removes blank details element
+      blankDetails.style.display = 'none';
+
+      //grabs first five moves of pokemon into an array
+      let pokemonMoves = res.data.moves
+        .slice(0, 5)
+        .map((move) => move.move.name);
+
+      //Add height to table
+      let pokeHeightValue = res.data.height;
+      let HeightInTable = document.getElementById('poke-height');
+      HeightInTable.innerHTML = pokeHeightValue + '"';
+
+      //Add weight to table
+      let pokeWeightValue = res.data.weight;
+      let WeightinTable = document.getElementById('poke-weight');
```

Comment:
The convention is to name variables using camel-case. 

```suggestion
      let weightInTable = document.getElementById('poke-weight');
```

Reply:
updated